website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -159,9 +210,12 @@ The following arguments are supported:
 
 The following arguments are supported:
 
-* `predefined_load_metric_specification` - (Optional) The load metric specification.
+* `predefined_load_metric_specification` - (Optional) The predefined load metric specification.
 * `predefined_metric_pair_specification` - (Optional) The metric pair specification from which Amazon EC2 Auto Scaling determines the appropriate scaling metric and load metric to use.
-* `predefined_scaling_metric_specification` - (Optional) The scaling metric specification.
+* `predefined_scaling_metric_specification` - (Optional) The predefined scaling metric specification.
+* `customized_scaling_metric_specification` - (Optional) The customized scaling metric specification. 
+* `customized_load_metric_specification` - (Optional) The customized load metric specification.
+* `customized_capacity_metric_specification` - (Optional) The customized capacity metric specification. The field is only valid when you use `customized_load_metric_specification`
```

Comment:
Can we have a link to the documentation for this? The API docs at https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PredictiveScalingCustomizedCapacityMetric.html does not mention this

Reply:
I think that's another miss on our public documentation. I would prefer to remove it to align with our current public doc.

Reply:
I agree, aligning with the public doc would be good. The Terraform doc can be updated in the future once the public AWS doc is updated